packages/aila/src/protocol/schema.ts

```diff
@@ -117,392 +115,312 @@ export const QuizOptionalSchema = z.array(QuizQuestionOptionalSchema);
+  title: dedent`The title of the lesson. Lesson titles should be a unique and succinct statement, 
+    not a question. Can include special characters if appropriate but should not use & sign instead 
+    of 'and'. Written in the TEACHER_TO_PUPIL_SLIDES voice. The title should be in sentence case 
+    starting with a capital letter and not end with a full stop. ${minMaxText({
+      max: 80,
+      entity: "characters",
+    })}`,
+  keyStage:
+    "The lesson's Key Stage as defined by UK educational standards. In slug format (kebab-case).",
+  subject:
+    "The subject that this lesson is included within, for instance English or Geography.",
+  topic:
+    "A topic that this lesson would sit within, which might cover several lessons with a shared theme.",
+  learningOutcome: dedent`What the pupils will have learnt by the end of the lesson. Should start 
+    with 'I can' and outline what pupils should be able to know/and or be able to do by the end of 
+    the lesson. Written in age appropriate language. May include a command word. Written in the PUPIL 
+    voice. ${minMaxText({ max: 190, entity: "characters" })}`,
+  learningCycles: dedent`An array of learning cycle outcomes. Should include a command word. Should 
+    be succinct. Should outline what pupils should be able to do/understand/know by the end of the 
+    learning cycle. Written in the TEACHER_TO_PUPIL_SLIDES voice. ${minMaxText({
+      min: 1,
+      max: 3,
+      entity: "elements",
+    })}`,
+  priorKnowledge: dedent`An array of prior knowledge statements, each being a succinct sentence. 
+    Written in the EXPERT_TEACHER voice. ${minMaxText({
+      min: 1,
+      max: 5,
+      entity: "elements",
+    })}`,
+  keyLearningPoints: dedent`An array of learning points, each being a succinct sentence. ${minMaxText(
+    {
+      min: 3,
+      max: 5,
+      entity: "elements",
+    },
+  )}`,
+  starterQuiz: dedent`The starter quiz for the lesson, which tests prior knowledge only, ignoring the content that is 
+    delivered in the lesson. Obey the rules as specified in the STARTER QUIZ section of the lesson plan 
+    guidance. Written in the TEACHER_TO_PUPIL_SLIDES voice.`,
+  exitQuiz: dedent`The exit quiz for the lesson, which tests the content that is delivered in the lesson. Written 
+    in the TEACHER_TO_PUPIL_SLIDES voice.`,
+  additionalMaterials:
+    "Any additional materials or notes that are required or useful for the lesson",
+} as const;
+
+export const LessonTitleSchema = z
+  .string()
+  .describe(LESSON_PLAN_DESCRIPTIONS.title);
+
+export const KeyStageSchema = z
+  .union([
+    z.enum([
+      "key-stage-1",
+      "key-stage-2",
+      "key-stage-3",
+      "key-stage-4",
+      "key-stage-5",
+      "early-years-foundation-stage",
+      "specialist",
+      "further-education",
+      "higher-education",
+    ]),
+    z.string(),
+  ])
+  .describe(LESSON_PLAN_DESCRIPTIONS.keyStage);
+
+export const SubjectSchema = z
```

Comment:
These are exported but currently unused. In a subsequent PR I'm planning on using these for edits to these sections, which are currently not present in the main prompt.